client/app/components/land-use-action.js

```diff
@@ -94,4 +94,16 @@ export default class LandUseActionComponent extends Component {
     this.args.pasForm[actionToRemove.attr1] = '';
     this.args.pasForm[actionToRemove.attr2] = '';
   }
+
+  @action
+  reconstructLandUseActionSelections () {
```

Comment:
Not too sure about this name... it's doing quite a lot here, and I think most of the things in this action could be used inside getters. In fact, once it's in getters, we probably won't need this action or the `did-insert` because we could use the getter in place of `this.landUseActionSelections` in the template:

```handlebars
{{#each this.answeredLandUseActions as |action|}}
  {{action}}
{{/each}}
```

Reply:
You're right, great suggestion!